recipe/meta.yaml

```diff
@@ -23,20 +23,29 @@ requirements:
 
 test:
   imports:
+    - opencensus
     - opencensus.common.runtime_context
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: https://github.com/census-instrumentation/opencensus-python
+  home: https://github.com/census-instrumentation/opencensus-python/tree/master/context/opencensus-context
   license: Apache-2.0
   license_file: LICENSE
   summary: The OpenCensus Runtime Context.
   description: |
     The OpenCensus Runtime Context provides in-process context propagation.
     By default, thread local storage is used for Python 2.7, 3.4 and 3.5;
     contextvars is used for Python >= 3.6, which provides asyncio support.
+    
+    PyPI: [https://pypi.org/project/opencensus-context/](https://pypi.org/project/opencensus-context/)
+    
   doc_url: https://github.com/census-instrumentation/opencensus-python/tree/master/context/opencensus-context/examples
   dev_url: https://github.com/census-instrumentation/opencensus-python/tree/master/context/opencensus-context
 
 extra:
   recipe-maintainers:
     - basnijholt
+    - sugatoray
```

Comment:
```suggestion
  recipe-maintainers:
    - basnijholt
    - dhirschfeld
    - sugatoray
```

Reply:
@dhirschfeld Thank you for troubleshooting it. :rocket: